perception/radar_fusion_to_detected_object/src/radar_object_fusion_to_detected_object_node/radar_object_fusion_to_detected_object_node.cpp

```diff
@@ -0,0 +1,218 @@
+    params.cbegin(), params.cend(),
+    [&name](const rclcpp::Parameter & p) { return p.get_name() == name; });
+
+  // Not found
+  if (itr == params.cend()) {
+    return false;
+  }
+
+  value = itr->template get_value<T>();
+  return true;
+}
+}  // namespace
+
+namespace radar_fusion_to_detected_object
+{
+using autoware_auto_perception_msgs::msg::DetectedObject;
+using autoware_auto_perception_msgs::msg::DetectedObjects;
+using autoware_auto_perception_msgs::msg::TrackedObjects;
+
+RadarObjectFusionToDetectedObjectNode::RadarObjectFusionToDetectedObjectNode(
+  const rclcpp::NodeOptions & node_options)
+: Node("radar_object_fusion_to_detected_object", node_options)
+{
+  // Parameter Server
+  set_param_res_ = this->add_on_set_parameters_callback(
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onSetParam, this, _1));
+
+  // Node Parameter
+  node_param_.update_rate_hz = declare_parameter<double>("node_params.update_rate_hz", 10.0);
+
+  // Core Parameter
+  core_param_.bounding_box_margin =
+    declare_parameter<double>("core_params.bounding_box_margin", 0.5);
+  core_param_.split_threshold_velocity =
+    declare_parameter<double>("core_params.split_threshold_velocity", 0.0);
+  core_param_.velocity_weight_average =
+    declare_parameter<double>("core_params.velocity_weight_average", 0.0);
+  core_param_.velocity_weight_median =
+    declare_parameter<double>("core_params.velocity_weight_median", 0.0);
+  core_param_.velocity_weight_target_value_average =
+    declare_parameter<double>("core_params.velocity_weight_target_value_average", 0.0);
+  core_param_.velocity_weight_target_value_top =
+    declare_parameter<double>("core_params.velocity_weight_target_value_top", 1.0);
+  core_param_.convert_doppler_to_twist =
+    declare_parameter<bool>("core_params.convert_doppler_to_twist", false);
+  core_param_.threshold_probability =
+    declare_parameter<float>("core_params.threshold_probability", 0.0);
+
+  // Core
+  radar_fusion_to_detected_object_ = std::make_unique<RadarFusionToDetectedObject>(get_logger());
+  radar_fusion_to_detected_object_->setParam(core_param_);
+
+  // Subscriber
+  sub_object_ = create_subscription<DetectedObjects>(
+    "~/input/objects", rclcpp::QoS{1},
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onDetectedObjects, this, _1));
+  sub_radar_ = create_subscription<TrackedObjects>(
+    "~/input/radars", rclcpp::QoS{1},
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onRadarObjects, this, _1));
+
+  // Publisher
+  pub_objects_ = create_publisher<DetectedObjects>("~/output/objects", 1);
+
+  // Timer
+  const auto update_period_ns = rclcpp::Rate(node_param_.update_rate_hz).period();
+  timer_ = rclcpp::create_timer(
+    this, get_clock(), update_period_ns,
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onTimer, this));
+}
+
+void RadarObjectFusionToDetectedObjectNode::onDetectedObjects(
+  const DetectedObjects::ConstSharedPtr msg)
+{
+  detected_objects_ = msg;
```

Comment:
Is there any reason not to use message_filters?

Reply:
I'll make new PR for message sync option.

Reply:
Can you describe the TODO in this PR?